faq.md

```diff
@@ -0,0 +1,157 @@
+
+<br/>
+
+Q: How can I flag specific datasets for public release?
+
+A: Within the dataset folder on Synapse, click Folder Tools -> Folder Sharing -> Make Public. This will make all subfolders and files within the folder publicly available. This feature is also available on individual files up to entire projects. Another option is to create a ‘public release’ fileview that shows all files in a tabular, queryable way rather than in folders.
+
+<br/>
+
+Q: Who do I contact for support?
+
+A: Reach out to your [center liaison](dcc-liaison). 
+
+<br/>
+
+Q: What is considered metadata?
+
+A: Metadata describes data that will have to be submitted with the data to be validated. For example, tumor type, tumor location, etc. 
+
+<br/>
+
+Q: Is there a dictionary for the metadata where I can explore?
+
+A:  You can reference the [HTAN Schema v12 Google Sheet](https://docs.google.com/spreadsheets/d/1tBK7lBcWU3FvdB2awtHLWbRuK6od3k4h60grDzydX8k/edit?usp=sharing) to explore the current metadata.
+
+<br/>
+
+Q:  Who is creating the data models?
+
+A: A metadata model is currently being built as a collaborative effort involving members of the various [HTAN Working Groups](https://www.synapse.org/#!Synapse:syn17022193/wiki/585904) and will continue to evolve.  Existing standards will be leveraged where suitable, e.g. HCA schemas for scRNA, GDC schema for clinical data. Extensions will be made to existing standards to fulfill HTAN specific needs.
+
+<br/>
+
+Q: How often will data models be released?
+
+A: Approximately once or twice a year, concurrent with the needs of data releases. 
+
+<br/>
+
+Q: How will spatial information be stored?
+
+A: Working groups are currently working on trying to define standards, please contact [HTAN Working Groups](https://www.synapse.org/#!Synapse:syn17022193/wiki/585904) if you have any comments or concerns. 
+
+<br/>
+
+Q: Can we have an intermediate data storage bucket? 
+
+A: Please contact your [DCC liaison](dcc-liaison) for consideration on a case by case basis. 
+
+<br/>
+
+Q: How can we use docker images/workflows/reproducible research practices? 
+
+A: You can use Synapse's internal docker registry, or you can link to a docker registry of your preference. Additionally, we will provide tools for linking your workflow to data you have access to on Synapse.
+
+<br/>
+
+## Data Uploader
+
+Q: How do I upload a \<dataset type> dataset?
+
+A: Please follow the standard operating procedures for data upload: [Data Ingress SOP](https://github.com/Sage-Bionetworks/HTAN-data-ingress-documentation-draft/blob/master/DOCUMENTATION.md).
+
+<br/>
+
+Q: What format does my data need to be in for submission? E.g do I need to organize my \<dataset type> dataset in a specific file/folder hierarchy?
+
+A: The topmost level folder needs to be descriptive because it will be used in the Data Ingress Service. Your data can be in any format or hierarchy under that folder. 
```

Comment:
would be good to link to the docs page for this!

Reply:
good call, just added!